hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -240,23 +240,19 @@
             key manager configuration enable.
             When key manager operation is started (see CONTROL), registers protected by this EN are no longer
             modifiable until the operation completes.
-
-            TBD
-            This should be enhanced to support multi-bit values in the future. Should be another script change.
           '''
         },
       ]
     },
 
-    { name: "CONTROL",
-      desc: "Key manager operation controls",
-      regwen: "CFG_REGWEN",
```

Comment:
do we need to use `regwen` for `start.en`? When happens if an operation starts and we program this CSR again?

Reply:
o fair point, it would not hurt. I think the way the logic is designed is that it would not matter, but no reason to take that chance. I'll add it.